eve/main.yml

```diff
@@ -10,18 +10,28 @@ stages:
     worker:
       type: local
     steps:
-      - TriggerStages:
-          name: Trigger build and lint stages simultaneously
-          stage_names:
-            - build
-            - lint
+    - TriggerStages:
```

Comment:
Is this new indentation yaml-valid?

Reply:
It's actually more valid than before - I didn't include Eve yaml in the yaml linting step since it uses some form of templating that's not directly YAML-valid, but the yaml linter _does_ yell on indentation quite a lot on this content taken from other `main.yml`s. Ditto with pod manifests.

Reply:
I ran `yamllint` to fix the indentation of the whole file (see 749652c)